specs/posix/auditd_fim_events.table

```diff
@@ -8,8 +8,10 @@ schema([
     Column("executable", TEXT, "The executable path"),
     Column("partial", TEXT, "True if this is a partial event (i.e.: this process existed before we started osquery)"),
     Column("cwd", TEXT, "The current working directory of the process"),
-    Column("path1", TEXT, "The path associated with the event"),
-    Column("path2", TEXT, "The canonical path associated with the event"),
+    Column("path", TEXT, "The path associated with the event"),
+    Column("dest_path", TEXT, "The canonical path associated with the event"),
```

Comment:
If this is the canonicalized version, why not call it `canonical`?

Reply:
They are both canonical paths; the *dest_path* is only used for rename, symlink and link syscall events